packages/react-client/src/hooks/useTrackManager.ts

```diff
@@ -132,6 +141,42 @@ export const useTrackManager = ({ mediaManager, tsClient }: TrackManagerConfig):
     mediaManager.enable();
   }
 
+  const stream = async () => {
+    if (joined.current) {
+      if (currentTrack?.trackId) {
+        await resumeStreaming();
+      } else {
+        await startStreaming();
+      }
+    }
+  };
+
+  async function toggle(mode: ToggleMode = "hard") {
+    const mediaStream = mediaManager.getMedia()?.stream;
+    const track =
```

Comment:
**suggestion:** `mediaManager.getTracks()[0]`

Reply:
![image](https://github.com/user-attachments/assets/e86cdafd-0d00-4dd4-bd36-d47c33254354)
![image](https://github.com/user-attachments/assets/a5a05f4c-71f3-4a2a-b9be-78307ab82412)
On my machine, the first one is always `audio`

Reply:
please have another look, i wrote `mediaManager.getTracks()`, not `mediaStream.getTracks()`

Reply:
![image](https://github.com/user-attachments/assets/7810fa26-4e42-4bea-8d77-686d27e85fcb)

Reply:
![image](https://github.com/user-attachments/assets/08ca0f18-464c-4614-b856-4422781179b2)